assets/wizards/dashboard/components/quick-actions.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Newspack - Dashboard, Quick Actions
+ *
+ * Quick Actions component provides editors quick access to content creation and viewing data relating to their site
+ */
+
+/**
+ * Dependencies
```

Comment:
This is a bit of a nit, but can we organize these following the import guidelines here: https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/#imports

We follow this pattern throughout the codebase so we should try to be consistent here as well.

i.e.

```
/**
 * WordPress dependencies
 */
import ...

/**
 * Internal dependencies
 */
import ...
```
```

Reply:
Appreciate the feedback, hope you don't mind but I went ahead and updated all dependency doc blocks in the IA feature to date - 975e908ea66533d71ff121670a1f3443f282e4a2.